system/lib/libc/musl/src/time/strftime.c

```diff
@@ -181,9 +181,9 @@ const char *__strftime_fmt_1(char (*s)[100], size_t *l, int f, const struct tm *
 			*l = 0;
 			return "";
 		}
-		*l = snprintf(*s, sizeof *s, "%+.2ld%.2d", // XXX EMSCRIPTEN: %d => %ld
+		*l = snprintf(*s, sizeof *s, "%+.2ld%.2ld", // XXX EMSCRIPTEN: %d => %ld
 			(tm->__tm_gmtoff)/3600,
-			abs(tm->__tm_gmtoff%3600)/60);
+			labs(tm->__tm_gmtoff%3600)/60); // XXX EMSCRIPTEN: abs => labs
```

Comment:
a warning that `abs` can't operate on `long` I believe.

Reply:
Maybe better to disable that warning rather than hack musl like this.  See:  https://github.com/emscripten-core/emscripten/blob/dfb2d09a6f4a3a3928e2b195ec3228aacd8f2f86/tools/system_libs.py#L682-L688

Also, it looks like this was removed upstream so this issue will go away once I get around to landing #13006: https://github.com/emscripten-core/musl/